src/apprt/gtk-ng/class/surface.zig

```diff
@@ -823,6 +858,41 @@ pub const Surface = extern struct {
             null,
             .{ .detail = "mouse-shape" },
         );
+
+        // Some other initialization steps
+        self.initUrlOverlay();
+    }
+
+    fn initUrlOverlay(self: *Self) void {
```

Comment:
Is it possible to add this to the Blueprint file or is that something that would come in a PR down the line?

Reply:
It doesn't seem possible to initialize overlays in a blueprint file. If we can, I'll do it!

Reply:
Oh also, its not possible to initialize event controllers in a blueprint/builder file. So, regardless of overlays, thats an issue. And again, if I'm wrong, I'd love to fix it. I couldn't find any docs on it, I read the source, and I just tried various thing, but none worked.